src/plugins/tables/tables.js

```diff
@@ -172,11 +173,12 @@ $document.on( "draw.dt", selector, function( event, settings ) {
 		)
 	) {
 		pagination.addClass( "hidden" );
+		pagination_top.addClass( "hidden" );
 	} else {
 
 		// Make sure Pagination is visible
 		pagination.removeClass( "hidden" );
-
+		pagination_top.removeClass( "hidden" );
 		// Update Pagination List
```

Comment:
Besoin d'une espace qui précède tout commentaire

```suggestion
		pagination_top.removeClass( "hidden" );

		// Update Pagination List
```